cibuild.sh

```diff
@@ -174,11 +174,11 @@ function run_builds {
   local ncpus=`grep -c ^processor /proc/cpuinfo`
   local options="-j $ncpus"
 
-  if [ "X$build" = "Xcheck" ]; then
```

Comment:
should we pass -x from command line instead of the hardcode relateship between full.data and -x?

Reply:
Good suggestion. Done